osdep/terminal-unix.c

```diff
@@ -514,6 +518,15 @@ void terminal_get_size2(int *rows, int *cols, int *px_width, int *px_height)
     *px_height = ws.ws_ypixel;
 }
 
+void terminal_show_cursor(bool visible)
+{
+    if (visible)
+        printf("\e[?25h");
+    else
+        printf("\e[?25l");
```

Comment:
Note: I read in the contributor document to always add braces, but this file doesn't, and I figured that the style of a particular file takes precedence over the general style.